autoload/maktaba/json.vim

```diff
@@ -1,8 +1,7 @@
 " Neovim support covered in https://github.com/neovim/neovim/issues/3417.
 " NOTE: Avoids using pre-1430 support for consistent Infinity/NaN.
-let s:HAS_NATIVE_JSON =
-    \ !has('nvim') &&
-    \ v:version > 704 || v:version == 704 && has('patch1430')
+let s:HAS_NATIVE_JSON = exists('*json_decode') && exists('v:null') &&
```

Comment:
Any particular reason you're checking for `v:null` as well? Or just because the existence of both implies that `json_decode()` is the built-in one?